alarm-frontend/src/Components/AlarmList.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Alarm from './Alarm';
+import AddAlarm from './AddAlarm';
+
+const BASE_URL = "http://127.0.0.1:5000"
+
+
+async function getAlarms() {
+    const alarms = await axios.get(`${BASE_URL}/alarms`);
+    return alarms.data;
+}
+let response = await getAlarms();
+
+function AlarmList() {
+    const [alarms, setAlarms] = useState([response])
+    let [count, setCount] = React.useState(0); 
+    const handleCount = ()=>{ 
+        setCount(++count);
+        console.log(count); 
+    }
+    
+    useEffect(function askAlarmAPI() {
+        async function getMoreAlarms() {
+            const alarms = await axios.get(`${BASE_URL}/alarms`);
+            setAlarms([alarms.data]);
+        }
+        getMoreAlarms();
+    }, [count])
+
+    return (
+        <div>
+            {alarms[0].map(({AMorPM, difficulty, hour, id, minute, type}) => (
```

Comment:
Oh looks like your alarms state is a list of alarm lists. Should just be an alarm list. Change `const [alarms, setAlarms] = useState([response])` to const [alarms, setAlarms] = useState([]) and have the useEffect set alarms to reaponse.data instead of setAlarms([alarms.data]);